content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+
+In MagnetoTellurics problems, the key diagnosed physical property is :ref:`electrical conductivity<electrical_conductivity_index>` :math:`\sigma`, as we expect the contrasts of the others physical properties to be negligible. 
+
+
+
+Setup
+-----
+
+ .. figure:: images/MT_N_layered_Earth-1.hires.png
+    :align: center
+    :scale: 50% 
+    :name: MTlayeredEarth
+
+    1D general Earth Model Configuration
+
+An example of a 1D MagnetoTelluric modelisation, with a 2-layers Earth, is shown in the movie above, where we have:
+
+ - a layered Earth, each layer with its own physical properties :math:`\sigma, \varepsilon, \mu`
+
+ - a plane wave traveling along the axis Z coming from air, composed of an electric field :math:`\mathbf{E_x}` and a magnetic field :math:`\mathbf{H_y}`. These are the fields we will be measuring with our geophysical instruments at the surface to obtain information from the underground
+
+
+We can see that several phenomenons are occuring. Just to mention few of them:
+
+ - the incoming wave (down component) is reflected at the surface
+
+ - once in the ground we observe a diffusive effect of the Earth on the wave that will regulate the depth of investigation and the resolution of the survey. The decay is more important in the second layer with a higher conductivity
+
+ - :math:`\mathbf{E_x}` and  :math:`\mathbf{H_y}` are continuous and phased compared to each other
+
+
+Governing Equations
+-------------------
+
+The governing equation for MagnetoTellurics problem can be obtained from
+:ref:`Maxwell's equations <maxwell1_fundamentals_index>`. We start with :ref:`Faraday's law<faraday>` and :ref:`Ampere's law<ampere_maxwell>`:
+
+.. math::
+    \nabla \times \mathbf{E_x} = - i \omega \mu \mathbf{H_y}
+    :label: Faraday
+    
+.. math::
+    \nabla \times \mathbf{H_y} = (\sigma + i \omega \varepsilon) \mathbf{E_x}
+    :label: Ampere
+
+Knowing that the divergent of **E** and **H** are equal to 0 here (no free charge)
+according to :ref:`Gauss's Law for Electric Fields<gauss_electric>` and :ref:`Gauss's Law for Magnetic Fields<gauss_magnetic_frequency>`, we can combine the equations to write the Helmhotz (wave propagation) equation for both **E** and **B** field:
+
+.. math::
+    \nabla ^2  \mathbf{E_x} + k^2 \mathbf{E_x} = 0
+    :label: E_wave_propagation_equation
+
+.. math::
+    \nabla ^2 \mathbf{H_y} + k^2 \mathbf{H_y} = 0
+    :label: H_wave_propagation_equation
+
+with k the wavenumber:
+
+.. math::
+    k = \sqrt{\omega ^2 \mu \varepsilon - i \omega \mu \sigma }
+    :label: kwavenumber
+
+
+In the ground, we usually assume that the displacement current is negligible, which means :math:`\sigma >> \omega \varepsilon`. In this case 
```

Comment:
I would also suggest clarifying the "we usually assume" (it is generally a safe assumption). i.e. "we can generally assume"